### PR TITLE
Merging Test Files and Target_Loader

### PR DESCRIPTION
**Pull request by chrishajduk84**

### CODE REVIEW

File: modules/csvcreator/CMakeLists.txt [hunk 1]
```diff
@@ -1,2 +1,6 @@
-include_directories(include)
+include_directories(include ../core/include)
+ADD_DEFINITIONS("-DBOOST_LOG_DYN_LINK")
 add_library(CSVCreator src/csv.cpp)
+target_link_libraries(CSVCreator ${OpenCV_LIBS} ${Boost_LIBRARIES} Core)
+target_compile_features(CSVCreator PRIVATE cxx_range_for)
```
Comment (benjaminwinger) [inline modules/csvcreator/CMakeLists.txt]: You're not using range for loops, this is unnecessary (same goes for the other modules, only TargetIdentification is actually using a range for loop).

**Comment by benjaminwinger** [comment]
In future (or even now if you want to try), when removing files such as the temporary files that were accidentally added to the commit, do it so that they're removed from the history as well, either by amending the original commit, or if there are commits added after the one you want to change, make a new commit, do an interactive rebase and fixup or squash the commit into the original commit (See [here](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) for an example). This will keep the history cleaner, and it can't be done cleanly once a fork has been merged. 

Also, I don't think that the test image I'd added for testing TargetIdentification is going to be terribly helpful for testing Volume Calculation. However what we really need to do it create a single resource folder to store data to be used in tests to avoid bloating the project's history. We should also perhaps configure it as a submodule so that you don't have to download the large test images when cloning the project.

File: modules/csvcreator/CMakeLists.txt (hunk 1 above)
Comment (chrishajduk84) [inline modules/csvcreator/CMakeLists.txt]: For whatever reason, the targetanalysis module needs it too.
When it compiles without the line, it errors during a file read.

**Comment by chrishajduk84** [comment]
Not sure what to rebase. I tried to merge 6b0c2d04049043feba33f58ba792dc5e816dd260 with e6c619ac8bdec5789e115530587e57dcba8215be...but there are a lot of commits in between, and I had problems doing so without messing with a large amount of conflicts and history.

**Comment by benjaminwinger** [comment]
I was talking about merging 70d6af8 with 7dbc33f, and that you should create a new commit and do the same to get rid of the unnecessary image files. 
Regarding the new commits you mentioned, personally I would just get rid of e6c619a entirely and rebase the branch onto upstream/master. For 6b0c2d0, note that I have got a boolean test class implemented [here](https://github.com/benjaminwinger/computer-vision/blob/test_update/modules/core/include/test.h) on a branch. just leave it for now though; the test_update branch isn't quite ready for merging.

**Comment by chrishajduk84** [comment]
Ah okay. I'll close this pull request then.

**Pull request closed without merging**